package.json

```diff
@@ -74,7 +75,7 @@
   "scripts": {
     "notes": "notes -s src/**",
     "sass-lint": "sass-lint -v -q",
-    "eslint": "eslint --ignore-path .gitignore .",
+    "eslint": "eslint --ignore-path .gitignore --fix .",
```

Comment:
do not change this task. create new one named `eslint:fix` and add fix flag there. We don't want auto fixing on CI.